tiledb/api/src/query/strategy/tests.rs

```diff
@@ -0,0 +1,1012 @@
+use cells::write::strategy::{WriteParameters, WriteSequenceParameters};
```

Comment:
Everything in this file was split out of `strategy.rs` up until `fn instance_query_condition`